packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -149,6 +148,32 @@ tbody.spectrum-Table-body {
     background-color: var(--spectrum-table-row-background-color-down);
   }
 
+  /* Alternative to border on rows. Using box shadow since they don't take room unlike border which would cause wiggles
+   * in the hightlight case and displace the sticky indicator. Also allows for a nicer bottom curved border to match the container,
+   * the bottom border curved corners were cut off when using borders.
+   */
+
+  /* Box shadow for bottom border for non-selected rows that aren't immediatly above a selected row. */
```

Comment:
Unfortunately cannot omit the bottom border of the last row like we do in ListView due to how the row vs cell is structured (aka row is 1px larger than the cell, controlled by layout, cell is responsible for filling in the background color, cell focus ring cut off if we reduce the row height, etc). Can be investigated further as follow up